conjure-java-core/src/test/resources/example-types.yml

```diff
@@ -222,6 +222,9 @@ types:
           listAlias: ListAlias
           setAlias: SetAlias
           mapAlias: MapAliasExample
+          booleanField:
```

Comment:
this is what was breaking in geotime

Reply:
https://github.palantir.build/geotime/geotime/blob/develop/geotime-api/src/main/conjure/geotime-search.yml#L108-L111